evm/src/fuzz/strategies/invariants.rs

```diff
@@ -75,7 +77,7 @@ fn generate_call(
             let senders = senders.clone();
             let fuzz_state = fuzz_state.clone();
             func.prop_flat_map(move |func| {
-                let sender = select_random_sender(senders.clone());
+                let sender = select_random_sender(fuzz_state.clone(), senders.clone());
```

Comment:
is this clone cheap?

Reply:
yes, it's an `Arc` clone, and we do use it in other places